apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -171,7 +171,7 @@ export class SpecialAnnouncementActionBlock extends Component {
   render() {
     return (
       <TwoColumnActionBlock
-        imageUrl={pegasus('/images/fill-540x289/teacher-apps-2.png')}
+        imageUrl={pegasus('/images/fill-540x289/teacher-apps-3.png')}
```

Comment:
I'm not familiar with this helper, but does this path need to start with `images/professional-learning/`?

Reply:
Oops. Yes!